### PR TITLE
Added dependencies for MockitoTest case

### PR DESCRIPTION
**Pull request by sophiaxu0424**
- Added dependencies for MockitoTest case
- Issue to: https://github.com/adoptium/aqa-tests/issues/5073

### CODE REVIEW

**Comment by llxia** [comment]
This PR should be merged together with https://github.com/adoptium/aqa-tests/pull/5133

File: scripts/getDependencies.pl [hunk 1]
```diff
@@ -94,6 +109,16 @@
 		fname => 'junit4.jar',
 		sha1 => 'e4f1766ce7404a08f45d859fb9c226fc9e41a861'
 	},
+	junit_jupiter_api => {
```
Comment (llxia) [inline scripts/getDependencies.pl:112]: This is not needed for PR https://github.com/adoptium/aqa-tests/pull/5133. Please remove.

**Review by karianna — APPROVED** [review]

**Review by llxia — APPROVED** [review]
LGTM

**Review by LongyuZhang — APPROVED** [review]
LGTM

**Comment by llxia** [comment]
https://openj9-jenkins.osuosl.org/job/test.getDependency/1072/

**Comment by llxia** [comment]
https://ci.adoptium.net/job/test.getDependency/

**Pull request merged**